src/main.py

```diff
@@ -0,0 +1,130 @@
+from typing import Optional
+
+import torch.optim
+import pandas as pd
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+from model import PitchSeqNN
+from src.data.dataset import ComposerClassificationDataset
```

Comment:
```suggestion
from model import PitchSeqNN
from data.dataset import ComposerClassificationDataset
```

This may require some changes to your module design, but I think it's better to not reference the `src` directory in code. Personally, would I would get rid of it completely, and `main.py`, `model.py` and `data/` could be root level.